indicators/NO_SLIT_001-004/R/NO_SLIT_001-004.qmd

```diff
@@ -0,0 +1,959 @@
+results.ANO.slit[['original']]
+# add columns for the indicator
+results.ANO.slit[['original']][,'tk'] <- NA
+results.ANO.slit[['original']][,'se'] <- NA
+results.ANO.slit[['original']][,'erosion'] <- NA
+results.ANO.slit[['original']][,'tk'] <-  as.numeric(results.ANO.slit[['original']][,'tk'])
+results.ANO.slit[['original']][,'se'] <- as.numeric(results.ANO.slit[['original']][,'se'])
+results.ANO.slit[['original']][,'erosion'] <- as.numeric(results.ANO.slit[['original']][,'erosion'])
+summary(results.ANO.slit[['original']])
+results.ANO.slit[['original']]$globalid <- as.factor(results.ANO.slit[['original']]$globalid)
+results.ANO.slit[['original']]$ano_flate_id <- as.factor(results.ANO.slit[['original']]$ano_flate_id)
+results.ANO.slit[['original']]$ano_punkt_id <- as.factor(results.ANO.slit[['original']]$ano_punkt_id)
+results.ANO.slit[['original']]$hovedoekosystem_250m2 <- as.factor(results.ANO.slit[['original']]$hovedoekosystem_250m2)
+results.ANO.slit[['original']]$hovedoekosystem_rute  <- as.factor(results.ANO.slit[['original']]$hovedoekosystem_rute)
+results.ANO.slit[['original']]$kartleggingsenhet_1m2 <- as.factor(results.ANO.slit[['original']]$kartleggingsenhet_1m2)
+results.ANO.slit[['original']]$hovedtype_rute    <- as.factor(results.ANO.slit[['original']]$hovedtype_rute)
+
+
+# roll out
+results.ANO.slit[['scaled']] <- results.ANO.slit[['original']]
+
+```
+
+```{r, eval = F, echo = F, warning = F}
+
+# calculating scaled values for the indicators based on the dataset
+for (i in 1:nrow(ANO.geo) ) {
+  tryCatch({
+    print(i)
+    print(paste(ANO.geo$ano_flate_id[i]))
+    print(paste(ANO.geo$ano_punkt_id[i]))
+
+    # scaling thresholds
+    ref <- 0
+    lim <- 1.5
+    maxmin <- 3.5
+    
+        # Disturbance evaluation for erosion
+    val <- ANO.geo$erosion[i]    
+    if ( !is.na(val) ) {
+      # coercing x into results.ANO dataframe
+      results.ANO.slit[["scaled"]][i,'erosion'] <- scal() 
+    }
+
+  }, error=function(e){cat("ERROR :",conditionMessage(e), "\n")})
+}
+
+#saveRDS(results.ANO.slit, paste0(here::here(),"/indicators/NO_SLIT_001-004/data/cache/results.ANO.slit.RDS"))
+
+```
+
+Now we have both variables on heavy vehicles and overuse scaled.
+
+```{r, include = F}
+# Use cache data
+results.ANO.slit<-readRDS(paste0(here::here(),"/indicators/NO_SLIT_001-004/data/cache/results.ANO.slit.RDS"))
+```
+
+
+#### Scaled value analyses
+
+In order to calculate regional indices for the indicator(s) in addition to national ones we need to add information on region to the results-object.
+```{r, message = F, echo = T, warning = F}
+#### preparing results data for calculating of indices ####
+## continuing with scaled
+res.ANO.slit <- results.ANO.slit[['scaled']]
+
+# add geometry again
+st_geometry(res.ANO.slit) <- st_geometry(ANO.geo)
+
+nor <- st_read(here::here("indicators/NO_SLIT_001-004/data/outlineOfNorway_EPSG25833.shp"),
+               quiet = T)%>%
+  st_as_sf() %>%
+  st_transform(crs = crs(ANO.geo))
```

Comment:
The code cracher here for me. Not sure where `crs()` comes from, but `st_crs()` works
```suggestion
  st_transform(crs = [crs](sf::st_crs)(ANO.geo))
```